### DISCUSSION TITLE
Voice implementer support for HTTP Signatures in IETF HTTP WG

### DISCUSSION THREAD

**Discussion by msporny** [mastodon/mastodon#27959] [General]
Hi, I'm @msporny, primary author of the HTTP Signatures specification at IETF for many years now. You've implemented some variation of that specification. 

I need your help to move that specification towards a global standard at IETF. Hearing from implementers, such as you, is a big part of determining if the work toward a global standard should proceed. The IETF HTTP Working Group is determining whether the work should proceed right now. This is very good news, because the European Banking API community, W3C DID Working Group, W3C Credentials Community Group and other standards setting organizations depend on implementations standardizing on a way to do HTTP Signatures.

The deadline for noting your support is Jan 31st 2020 (in ~10 days).

Here's where you can make a difference...

Here is the IETF HTTP WG Call for Adoption:

https://lists.w3.org/Archives/Public/ietf-http-wg/2020JanMar/0002.html

To note your support of the specification:

1. Go here and click "subscribe to this list": https://lists.w3.org/Archives/Public/ietf-http-wg/
2. Verify your subscription by checking your email and clicking on the link that is mailed to you.
3. Go here and click "respond to this message": https://lists.w3.org/Archives/Public/ietf-http-wg/2020JanMar/0002.html
4. Write an email stating:
  4.1 That you support the adoption of the draft.
  4.2 Why you support the adoption of the draft.
  4.3 How you plan to make use the specification, either directly, or indirectly (via someone else's software).
5. Set up an email filter to put all mail sent to ietf-http-wg@w3.org into its own folder. The mailing list averages ~350 emails/month. You can also leave the mailing list immediately after sending the email above if that amount of email traffic is unacceptable to you.

For an example of the type of email you could write, see this:

https://lists.w3.org/Archives/Public/ietf-http-wg/2020JanMar/0018.html

Thanks a ton for supporting the specification through your implementation. I hope you consider helping us take the specification across the goal line by voicing your support in the IETF HTTP Working Group!

**Comment by Gargron** [comment]
Is it not possible to respond to the thread if you use webmail?

**Comment by msporny** [comment]
> Is it not possible to respond to the thread if you use webmail?

Hey @Gargron, unfortunately no... BUT, if you just use a subject line like this in your Webmail client: "Re: Call for Adoption: draft-richanna-http-message-signatures", that will work as well.

Another option is to just respond in this issue and I can forward your input into the mailing list (and link to this issue). This is time sensitive - the call for adoption ends at the end of today. Please try to respond before the end of today 11pm European time (so I have time to forward your response before midnight).
